Todo.Application/Class1.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Todo.Application
+{
+    public class Class1
```

Comment:
Please check, if this class is not being used then remove it. Same goes for all such classes that are created by default by IDE and are not being used

Reply:
Done